.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,60 @@
+name: Docker Image CI
+on:
+  push:
+    tags:
+      - v*
+  pull_request:
+    types:
+      - closed
```

Comment:
`closed` is merged?

Reply:
No, it's closed. Although lated on lines 14-15 we add a rule only for merged PR.
``` 
  if_merged:
    if: github.event.pull_request.merged == true
```

Reply:
but if it runs only on `closed` PRs with a rull on `merged` ones - it will never run ?

Reply:
or `closed` inclues `merged`?